website/blog/2025/05-19-enhanced-node-management-introducing-in-place-updates-in-gardener.md

```diff
@@ -73,12 +73,7 @@ In-place node updates represent a significant step forward in Gardener's operati
 ### Dive Deeper
 
 To explore the technical details and contributions that made this feature possible, refer to the following resources:
-*   **GEP-31 API Changes**: [PR #11191](https://github.com/gardener/gardener/pull/11191)
-*   **OSC Adaptations**: [PR #11393](https://github.com/gardener/gardener/pull/11393)
-*   **MCM API Changes**: [PR #11631](https://github.com/gardener/gardener/pull/11631)
-*   **Worker Reconciler Adaptations**: [PR #11713](https://github.com/gardener/gardener/pull/11713)
-*   **Gardener Node Agent Adaptations**: [PR #11718](https://github.com/gardener/gardener/pull/11718)
-*   **Shoot Reconciler Adaptations**: [PR #11843](https://github.com/gardener/gardener/pull/11843)
-*   **Shoot Status Reconciler**: [PR #11844](https://github.com/gardener/gardener/pull/11844)
-*   **Controller Optimizations & E2E Tests**: [PR #11953](https://github.com/gardener/gardener/pull/11953)
+*   **Parent Issue for "[GEP-31] Support for In-Place Node Updates
+" **: [Issue #10219](https://github.com/gardener/gardener/issues/10219 
+*   **GEP-31: In-Place Node Updates of Shoot Clusters**: [https://github.com/gardener/gardener/blob/master/docs/proposals/31-inplace-node-update.md]
```

Comment:
```suggestion
*   **Parent Issue for "[GEP-31] Support for In-Place Node Updates
" **: [Issue #10219](https://github.com/gardener/gardener/issues/10219)
*   **GEP-31: In-Place Node Updates of Shoot Clusters**: [https://github.com/gardener/gardener/blob/master/docs/proposals/31-inplace-node-update.md](https://github.com/gardener/gardener/blob/master/docs/proposals/31-inplace-node-update.md)
```
My bad, it should've been like this.
Do you think putting the link directly looks out-of-place? If yes, maybe you can update it in a better way.

Reply:
Resolved with my latest commit. I think that we should avoid using direct links as link text and replace them appropriately - either use the topic title, Issue number or platform where the media is based.

I might go over all the blogs at some point in order to unify the way these links are used.